[Bug] How to build

### Cloudberry Database and PL/Java version

main branch from git

### What happened

When running make i get:
```
curl -O https://artifactory.hashdata.xyz/artifactory/greenplum/cbdb-build-dependencies/nar-maven-plugin.tar.gz
curl: (7) Failed to connect to artifactory.hashdata.xyz port 443 after 75880 ms: Couldn't connect to server
```

### What you think should happen instead

curl should succeed

### How to reproduce

Run ``make``

### Operating System

Ubuntu focal

### Anything else

_No response_

### Are you willing to submit PR?

- [ ] Yes, I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://cloudberrydb.org/community/coc).

Hi,

i tried again with updated source code.

Now the error is:

```
[INFO] Installing /usr/src/mymaven/nar-maven-plugin/target/nar-maven-plugin-3.5.2.jar to /root/.m2/repository/com/github/maven-nar/nar-maven-plugin/3.5.2/nar-maven-plugin-3.5.2.jar
[INFO] Installing /usr/src/mymaven/nar-maven-plugin/pom.xml to /root/.m2/repository/com/github/maven-nar/nar-maven-plugin/3.5.2/nar-maven-plugin-3.5.2.pom
[INFO] ------------------------------------------------------------------------
[INFO] BUILD SUCCESS
[INFO] ------------------------------------------------------------------------
[INFO] Total time:  10.172 s
[INFO] Finished at: 2024-09-06T06:36:44Z
[INFO] ------------------------------------------------------------------------
mvn clean install
[INFO] Scanning for projects...
[WARNING] The POM for com.github.maven-nar:nar-maven-plugin:jar:3.10.2-SNAPSHOT is missing, no dependency information available
[ERROR] [ERROR] Some problems were encountered while processing the POMs:
[ERROR] Unresolveable build extension: Plugin com.github.maven-nar:nar-maven-plugin:3.10.2-SNAPSHOT or one of its dependencies could not be resolved:
	Could not find artifact com.github.maven-nar:nar-maven-plugin:jar:3.10.2-SNAPSHOT
 @
[ERROR] Unknown packaging: nar @ line 14, column 13
 @
[ERROR] The build could not read 1 project -> [Help 1]
[ERROR]
[ERROR]   The project org.postgresql:pljava-so:1.5.0 (/usr/src/mymaven/pljava-so/pom.xml) has 2 errors
[ERROR]     Unresolveable build extension: Plugin com.github.maven-nar:nar-maven-plugin:3.10.2-SNAPSHOT or one of its dependencies could not be resolved:
[ERROR]     	Could not find artifact com.github.maven-nar:nar-maven-plugin:jar:3.10.2-SNAPSHOT
[ERROR]     -> [Help 2]
[ERROR]     Unknown packaging: nar @ line 14, column 13
[ERROR]
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR]
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/ProjectBuildingException
[ERROR] [Help 2] http://cwiki.apache.org/confluence/display/MAVEN/PluginManagerException
make: *** [Makefile:50: build] Error 1
```

Kind Regards,

David

--- a/pljava-so/pom.xml
+++ b/pljava-so/pom.xml
@@ -49,6 +49,7 @@
 					<plugin>
 						<groupId>com.github.maven-nar</groupId>
 						<artifactId>nar-maven-plugin</artifactId>
+						<version>3.10.1</version>
 						<configuration>
 							<linker>
 								<libs combine.children='append'>
@@ -185,7 +186,7 @@
 					<plugin>
 						<groupId>com.github.maven-nar</groupId>
 						<artifactId>nar-maven-plugin</artifactId>
-						<version>3.10.2-SNAPSHOT</version>
+						<version>3.10.1</version>
 						<configuration>
 							<c>
 								<options>
@@ -220,7 +221,7 @@
 					<plugin>
 						<groupId>com.github.maven-nar</groupId>
 						<artifactId>nar-maven-plugin</artifactId>
-						<version>3.10.2-SNAPSHOT</version>
+						<version>3.10.1</version>
 						<configuration>
 							<linker>
 								<options combine.children='append'>
@@ -304,7 +305,7 @@
 			<plugin>
 				<groupId>com.github.maven-nar</groupId>
 				<artifactId>nar-maven-plugin</artifactId>
-				<version>3.10.2-SNAPSHOT</version>
+				<version>3.10.1</version>
 				<extensions>true</extensions>
 
 				<configuration>
@@ -409,7 +410,7 @@
 									<pluginExecutionFilter>
 										<groupId>com.github.maven-nar</groupId>
 										<artifactId>nar-maven-plugin</artifactId>
-										<version>3.10.2-SNAPSHOT</version>
+										<version>3.10.1</version>
 										<goals>
 											<goal>nar-compile</goal>
 											<goal>nar-download</goal>

